composer.json

```diff
@@ -6,8 +6,8 @@
     "require": {
         "ext-dom": "*",
         "ext-simplexml": "*",
-        "ondrejmirtes/better-reflection": "^4.3.19",
         "php": ">=7.3",
+        "phpstan/phpstan": "0.12.98",
```

Comment:
We are adding a too big tool into require 
Instead, we should fix the validation tool. 
Dependencies were correct.